### ISSUE TITLE
Support for 5.1 surround sound

### ISSUE THREAD

**Issue by kcajjones** [yuliskov/SmartTubeNext#2235]
YouTube official app supports 5.1 surround sound on my Sony X900h TV.

The same videos don't offer an option for 5.1 audio on the STN app.

**Comment by SodaWithoutSparkles** [comment]
Example video please?

**Comment by kcajjones** [comment]
I can confirm that these two videos have a 5.1 surround button available on the gui that is automatically enabled on my TV. I have a Denon x3300w connected via hdmi to an earc adapter to my Sony x900h.

https://youtu.be/fYZuiFDQwQw

https://youtu.be/nIzOAm-qZGQ

**Comment by fawzib** [comment]
smarttubenext does play them correctly but it never selects them by default even thought the system supports it. might be cool to allow an audio preset as well.

**Comment by ehsope** [comment]
Pretty sure it defaulted to the surround track a year ago, but that has changed. If we could set a preference list (ie DD+ > DD > Opus) that would be great.

**Comment by ahmetyilmaz** [comment]
I am not even able to select surround sound on newest Chromecast. Other apps and official YT app sounds fine but SmartTube nevers works.

**Comment by inertia666** [comment]
Yeah surround sound is not defaulting for surround videos. Would be great if that was fixed.
